Topic 0/src/abstractFactory/MySqlFactory.java

```diff
@@ -0,0 +1,14 @@
+package abstractFactory;
+
+public class MySqlFactory implements DatabaseFactory{
+
+	@Override
+	public Database getDatabase(int id) {
+		switch (id) {
+		case 1: return new MySqlOne();
+		case 2: return new MySqlTwo();
+		default: return new MySqlOne();
```

Comment:
Onstead of returning a default database for a wrong # for example let's say you pass in a 123 by error you should get an exception not a default MySql as if your input were ok.